examples/lookup/src/company-details.js

```diff
@@ -26,5 +26,8 @@ module.exports.post = function (req, res) {
     }
   } else {
     res.setStatusCode(400)
+      .setPayload({
+        'message': 'This is my custom friendly error message that will be shown to the user on a failed lookup'
+      })
```

Comment:
Can we add a comment above here to outline that custom error messages are only shown if the response http status code is 400 and the payload is in this format.